### PR TITLE
Stop using short open tags (resolves VUFIND-1257).

### PR DESCRIPTION
**Pull request by demiankatz**
TODO
- [x] Discuss whether this is the best option, and whether we should clean up any formatting
- [x] Add note to changelog

### CODE REVIEW

**Comment by demiankatz** [comment]
I am very puzzled by the linting failures that Travis is reporting across all PHP versions. I cannot reproduce these problems on my local server. Brilliant ideas welcomed.

File: themes/bootstrap3/templates/Auth/MultiILS/loginfields.phtml [hunk 1]
```diff
@@ -4,7 +4,7 @@
   <select id="login_target" name="target" class="form-control">
     <?foreach ($this->auth()->getManager()->getLoginTargets() as $target):?>
```
Comment (xmorave2) [inline themes/bootstrap3/templates/Auth/MultiILS/loginfields.phtml]: Demian, could be this the problem? The opening tag is not followed by space and then not catched by this patch.

Comment (xmorave2) [reply themes/bootstrap3/templates/Auth/MultiILS/loginfields.phtml]: I just looked into the first file reported by Travis, the other files could have the same or other problem ;)

**Comment by EreMaijala** [comment]
@demiankatz Travis php has short_open_tag disabled, and disabling them locally allowed me to also get the same results as Travis did.

**Comment by demiankatz** [comment]
Good eyes, @xmorave2 and @EreMaijala. That makes perfect sense. We're closed today due to winter weather, but I'll get this sorted out when I return. Thanks!

**Comment by EreMaijala** [comment]
I pushed a change to increase the verbosity of errors from php-cs-fixer. Maybe it'll help diagnose any issues in the future.

**Comment by demiankatz** [comment]
@EreMaijala, thanks for the verbosity increase, that will be helpful in future -- I've moved it over to master. I've also fixed the missing spaces @xmorave2 noticed in master, and rebuilt this branch based on that cleaned-up code. I think we're in much better shape now. Some of the templates are a bit uglier with the full php tags, but I think we'll have to live with that for the benefits it provides us. Let me know if you have any further thoughts on this -- we'll likely discuss it in more detail on next week's developers call.

**Pull request merged**